src/control/ros/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#==============================================================================
+# Dependencies
+#
+find_package(trajectory_msgs REQUIRED)
+aikido_check_package(trajectory_msgs "aikido::control::ros" "trajectory_msgs")
```

Comment:
The `control/ros` namespace is intended to be optional, so `traectory_msgs` should *not* be marked as `REQUIRED`. We should change this to:
```
find_package(trajectory_msgs QUIET)
aikido_check_package(geometry_msgs "control::ros" "trajectory_msgs")
```